### PR TITLE
Ypp landing page

### LINKED ISSUE #3149: YT LP Static Content
## Scope

- [ ] Implement all static content
- [ ] CTAs Claim your channel and Authorise with Youtube start the same Authentication journey

Out of scope:
- [ ] Auth Journey

## Design

[Link to Figma > ](https://www.figma.com/file/oQqFqdAiPu16eeE2aA5AD5/YouTube-Partner-Program?node-id=1604%3A85511)

Screenshot 2022-08-18 at 20 52 36

### PR DESCRIPTION
**Pull request by drillprop**
Fix #3149

### CODE REVIEW

File: packages/atlas/src/components/LayoutGrid/LayoutGrid.tsx [hunk 1]
```diff
@@ -8,10 +8,12 @@ type ReponsivenessObject = Partial<Record<keyof typeof media | 'base', number>>
...
   display: grid;
   grid-template-columns: repeat(12, 1fr);
-  grid-gap: 16px;
+  row-gap: 16px;
+  column-gap: 16px;
 
   ${media.md} {
-    grid-gap: 24px;
+    row-gap: 24px;
+    column-gap: 24px;
```
Comment (rafalpawlow) [inline packages/atlas/src/components/LayoutGrid/LayoutGrid.tsx]: `sizes` pls

File: packages/atlas/src/views/global/YppLandingView/YppLandingView.tsx [hunk 2]
```diff
@@ -25,33 +66,276 @@ export const YppLandingView: FC = () => {
...
+        <StyledLimitedContainerWidth as="section">
+          {/* TODO add reward section above */}
+          <CenteredLayoutGrid>
+            <HeaderGridItem
+              marginBottom={8}
+              as="header"
+              colStart={{ sm: 3, lg: 4 }}
+              colSpan={{ base: 12, sm: 8, lg: 6 }}
+            >
+              <Text variant={mdMatch ? 'h800' : 'h600'} as="h2">
```
Comment (rafalpawlow) [inline packages/atlas/src/views/global/YppLandingView/YppLandingView.tsx]:
```suggestion
              <Text variant={mdMatch ? 'h700' : 'h600'} as="h2">
```

File: packages/atlas/src/views/global/YppLandingView/YppLandingView.tsx [hunk 3]
```diff
@@ -25,33 +66,276 @@ export const YppLandingView: FC = () => {
...
+        <StyledLimitedContainerWidth as="section">
+          <CardsWithImagesContainer>
+            <CenteredLayoutGrid>
+              <HeaderGridItem
+                marginBottom={mdMatch ? 24 : 16}
+                as="header"
+                colStart={{ sm: 3, lg: 4 }}
+                colSpan={{ base: 12, sm: 8, lg: 6 }}
+              >
+                <Text variant={mdMatch ? 'h800' : 'h600'} as="h2">
```
Comment (rafalpawlow) [inline packages/atlas/src/views/global/YppLandingView/YppLandingView.tsx]:
```suggestion
                <Text variant={mdMatch ? 'h700' : 'h600'} as="h2">
```

File: packages/atlas/src/views/global/YppLandingView/YppLandingView.tsx [hunk 4]
```diff
@@ -25,33 +66,276 @@ export const YppLandingView: FC = () => {
...
+              <GridItem colStart={{ sm: 2, md: 1, lg: 2 }} colSpan={{ base: 12, sm: 10, md: 7, lg: 6 }}>
+                <ImageContainer>
+                  <CardImage dropShadow absolute src={dashboardImgBack} alt="Dashboard" />
+                  <CardImage dropShadow src={dashboardImgFront} ref={dashboardImageRef} alt="Reward card" />
+                </ImageContainer>
+              </GridItem>
+              <GridItem colStart={{ sm: 3, md: 8 }} colSpan={{ base: 12, sm: 8, md: 5, lg: 4 }}>
+                <ContentCard
+                  title="Manage your channel on a simple dashboard"
+                  subtitle="YPP dashboard"
```
Comment (rafalpawlow) [inline packages/atlas/src/views/global/YppLandingView/YppLandingView.tsx]:
```suggestion
                  subtitle="Dashboard"
```

File: packages/atlas/src/views/global/YppLandingView/YppLandingView.styles.ts [hunk 5]
```diff
@@ -38,3 +49,186 @@ export const BackImage = styled.img`
...
+  ${media.md} {
+    background-image: url(${bottomLeftPattern});
+    background-position: bottom left;
+    background-repeat: no-repeat;
+  }
+`
+
+export const CardsWithImagesContainer = styled.div`
+  display: grid;
+  gap: ${sizes(16)};
```
Comment (rafalpawlow) [inline packages/atlas/src/views/global/YppLandingView/YppLandingView.styles.ts]: From MD it should be `24`
Zrzut ekranu 2022-09-8 o 15 56 46

File: packages/atlas/src/views/global/YppLandingView/YppLandingView.styles.ts [hunk 6]
```diff
@@ -38,3 +49,186 @@ export const BackImage = styled.img`
...
+
+  ${({ dropShadow }) => dropShadow && imageShadow};
+`
+
+export const StyledLimitedContainerWidth = styled(LimitedWidthContainer)`
+  padding-bottom: unset;
+`
+
+export const CtaBanner = styled.div`
+  padding: ${sizes(16)} ${sizes(8)};
```
Comment (rafalpawlow) [inline packages/atlas/src/views/global/YppLandingView/YppLandingView.styles.ts]: From MD it should be `${sizes(24)} ${sizes(12)}`

File: packages/atlas/src/views/global/YppLandingView/YppLandingView.tsx [hunk 7]
```diff
@@ -25,33 +66,276 @@ export const YppLandingView: FC = () => {
...
+                  title="New ways to engage and get support from your audience"
+                  subtitle="CREATOR TOKENS"
+                  body="Turn viewers into evengelists [?] by letting them hold a share in your channel when they buy your own channel token."
+                />
+              </GridItem>
+            </CardImageRow>
+          </CardsWithImagesContainer>
+        </StyledLimitedContainerWidth>
+      </BackgroundContainer>
+      <BackgroundContainer as="footer" noBackground>
```
Comment (rafalpawlow) [inline packages/atlas/src/views/global/YppLandingView/YppLandingView.tsx]: This element should have different padding  from SM  than other `BackgroundContainer` 

Zrzut ekranu 2022-09-8 o 16 06 27

File: packages/atlas/src/views/global/YppLandingView/YppLandingView.tsx [hunk 8]
```diff
@@ -25,33 +66,276 @@ export const YppLandingView: FC = () => {
...
+              </GridItem>
+              <GridItem
+                colStart={{ sm: 3, md: 1, lg: 2 }}
+                rowStart={{ md: 1 }}
+                colSpan={{ base: 12, sm: 8, md: 5, lg: 4 }}
+              >
+                <ContentCard
+                  title="Backup your videos and own them forever"
+                  subtitle="YouTube backup"
+                  body="You can sleep peacefully knowing that your conent is safe and forever stored on blockchain. You can even sell it as NFTs — you own it forever."
```
Comment (rafalpawlow) [inline packages/atlas/src/views/global/YppLandingView/YppLandingView.tsx]:
```suggestion
                  body="You can sleep peacefully knowing that your content is safe and forever stored on blockchain. You can even sell it as NFTs — you own it forever."
```

File: packages/atlas/src/views/global/YppLandingView/YppLandingView.tsx [hunk 9]
```diff
@@ -25,33 +66,276 @@ export const YppLandingView: FC = () => {
...
+                </ImageContainer>
+              </GridItem>
+              <GridItem colStart={{ sm: 3, md: 8 }} colSpan={{ base: 12, sm: 8, md: 5, lg: 4 }}>
+                <ContentCard
+                  pill={{
+                    label: 'Coming later this year',
+                  }}
+                  title="Automatic YouTube video sync"
+                  subtitle="YouTube sync"
+                  body="Set up once and do nothing. Your videos will be automaticly synced from youtube to Atlas and you will still earn for it passively."
```
Comment (rafalpawlow) [inline packages/atlas/src/views/global/YppLandingView/YppLandingView.tsx]: In some places we write Youtube with a capital letter and sometimes we don't, probably to consult with designers.

File: packages/atlas/src/views/global/YppLandingView/YppLandingView.tsx [hunk 10]
```diff
@@ -25,33 +66,276 @@ export const YppLandingView: FC = () => {
...
+                    alt="Context menu of nft tile"
+                  />
+                  <CardImage src={videoNfts4} ref={nftCursorImageRef} alt="Cursor" />
+                </ImageContainer>
+              </GridItem>
+              <GridItem colStart={{ sm: 3, md: 8 }} colSpan={{ base: 12, sm: 8, md: 5, lg: 4 }}>
+                <ContentCard
+                  title="Monetise your videos with NFT sales and royalties"
+                  subtitle="Video NFTs"
+                  body="Turn your videos into NFTs and put them up for sale.  Choose between fixed price, open auction, and timed auction. Define royalties to get a cut every time your NFT gets sold again.There you can check your channel status and get instructions on completing all actions to get rewards"
```
Comment (rafalpawlow) [inline packages/atlas/src/views/global/YppLandingView/YppLandingView.tsx]:
```suggestion
                  body="Turn your videos into NFTs and put them up for sale. Choose between fixed price, open auction, and timed auction. Define royalties to get a cut every time your NFT gets sold again. There you can check your channel status and get instructions on completing all actions to get rewards"
```

**Review by rafalpawlow — CHANGES_REQUESTED** [review]
1. These cards are broken on SM, I cannot scroll to see 1st one
Zrzut ekranu 2022-09-8 o 15 28 07

2. I don't like the position of this context menu image, in figma it's clearly visible that kebab button is clicked. Also it's shifted to the left comparing to figma.

yours:
Zrzut ekranu 2022-09-8 o 16 11 00

figma:
Zrzut ekranu 2022-09-8 o 16 11 58

File: packages/atlas/src/views/global/YppLandingView/YppLandingView.tsx (hunk 4 above)
Comment (drillprop) [reply packages/atlas/src/views/global/YppLandingView/YppLandingView.tsx]: Just found we have different copies in a few places
image
image
That's why there are many mistakes.

I think I'll use [this frame](https://www.figma.com/file/oQqFqdAiPu16eeE2aA5AD5/YouTube-Partner-Program?node-id=1606%3A85515) as source of truth

File: packages/atlas/src/views/global/YppLandingView/YppLandingView.tsx (hunk 8 above)
Comment (drillprop) [reply packages/atlas/src/views/global/YppLandingView/YppLandingView.tsx]: I used a different copy after all - based on [this frame](https://www.figma.com/file/oQqFqdAiPu16eeE2aA5AD5/YouTube-Partner-Program?node-id=1606%3A85515)

File: packages/atlas/src/views/global/YppLandingView/YppLandingView.tsx (hunk 9 above)
Comment (drillprop) [reply packages/atlas/src/views/global/YppLandingView/YppLandingView.tsx]: In [this frame](https://www.figma.com/file/oQqFqdAiPu16eeE2aA5AD5/YouTube-Partner-Program?node-id=1606%3A85515) only `YouTube` is used. I don't think it's worth consulting this :P

**Comment by drillprop** [comment]
@rafalpawlow I tweaked the parallax effect a bit, but I'd opt to not focus on this too much right now. Let's wait for Adam and then we could revisit and do more improvements. WDYT?

**Review by rafalpawlow — CHANGES_REQUESTED** [review]
Something is wrong with parallax, it's not smooth anymore

https://user-images.githubusercontent.com/13706246/189613218-72eb171d-26b9-4e26-b1bc-bf1d50e14301.mov

**Review by rafalpawlow — CHANGES_REQUESTED** [review]
There is some unnecessary margin under paragraph
Zrzut ekranu 2022-09-12 o 10 58 56

**Comment by drillprop** [comment]
> Something is wrong with parallax, it's not smooth anymore
> 

I can't reproduce

**Comment by drillprop** [comment]
> There is some unnecessary margin under paragraph

Fixed.
Regarding parallax - let's discuss this on tomorrow's dev sync. Maybe it's worth replacing the parallax library.

**Comment by kdembler** [comment]
I've tested on mobile Safari and works poorly :( A lot of images are not loaded at all, and the parallax doesn't look great. I think we should consider disabling parallax for mobile, it may fix some issues:

https://user-images.githubusercontent.com/12646744/189834809-8fce95e6-d0b3-4652-91ed-73020cee358c.MP4

**Comment by drillprop** [comment]
@rafalpawlow unfortunately, you'll need to go through my code once again. I moved all the sections to separate files and did some refactoring. 
Performance issues should be fixed. I left the code responsible for dropping shadows but commented it. This is because I think we could use something else here(like box-shadow), but I'll need to consult this with Adam first.

**Review by rafalpawlow — APPROVED** [review]

**Review by rafalpawlow — APPROVED** [review]

**Pull request merged**